translate/automl/automl_translation_model.py

```diff
@@ -0,0 +1,298 @@
+    print("Model display name: {}".format(model.display_name))
+    print("Model create time:")
+    print("\tseconds: {}".format(model.create_time.seconds))
+    print("\tnanos: {}".format(model.create_time.nanos))
+    print("Model deployment state: {}".format(deployment_state))
+
+
+# [END automl_translation_get_model]
+
+
+# [START automl_translation_list_model_evaluations]
+def list_model_evaluations(project_id, compute_region, model_id, filter_):
+    """List model evaluations.
+    Args:
+        project_id: Id of the project.
+        compute_region: Region name.
+        model_id: Id of the model.
+        filter_: Filter expression.
+    """
+    client = automl.AutoMlClient()
+
+    # Get the full path of the model.
+    model_full_id = client.model_path(project_id, compute_region, model_id)
+
+    print("List of model evaluations:")
+    for element in client.list_model_evaluations(model_full_id, filter_):
+        print(element)
+
+
+# [END automl_translation_list_model_evaluations]
+
+
+# [START automl_translation_get_model_evaluation]
+def get_model_evaluation(
+    project_id, compute_region, model_id, model_evaluation_id
+):
+    """Get model evaluation.
+    Args:
+        project_id: Id of the project.
+        compute_region: Region name.
+        model_id: Id of the model.
+        model_evaluation_id: Id of your model evaluation.
+    """
+    client = automl.AutoMlClient()
+
+    # Get the full path of the model evaluation.
+    model_evaluation_full_id = client.model_evaluation_path(
+        project_id, compute_region, model_id, model_evaluation_id
+    )
+
+    # Get complete detail of the model evaluation.
+    response = client.get_model_evaluation(model_evaluation_full_id)
+
+    print(response)
+
+
+# [END automl_translation_get_model_evaluation]
+
+
+# [START automl_translation_delete_model]
+def delete_model(project_id, compute_region, model_id):
+    """Delete a model.
+    Args:
+        project_id: Id of the project.
+        compute_region: Region name.
+        model_id: Id of the model.
+    """
+    client = automl.AutoMlClient()
+
+    # Get the full path of the model.
+    model_full_id = client.model_path(project_id, compute_region, model_id)
+
+    # Delete a model.
+    response = client.delete_model(model_full_id)
+
+    # synchronous check of operation status.
+    print("Model deleted. {}".format(response.result()))
+
+
+# [END automl_translation_delete_model]
+
+
+# [START automl_translation_get_operation_status]
+def get_operation_status(operation_full_id):
+    """Get operation status.
+    Args:
+        operation_full_id: Full name of a operation.
+            For example, the name of your operation is
+            projects/<projectId>/locations/us-central1/operations/<operationId>
+    """
+    client = automl.AutoMlClient()
+
+    # Get the latest state of a long-running operation.
+    response = client.transport._operations_client.get_operation(

```

Comment:
Can you explain why you need to be able to get the operation status? If you think there is a legitimate use case for this we may want to consider adding a method on the client object for this, since you are relying on an internal object here.

Reply:
these training jobs can be long running, up to a day.

Reply:
Alright. I am going to create an issue on the library for us to consider this as a change. Thanks!

Reply:
I am not 100% sure but I think response.operation.refresh() will give the current status.

Reply:
what would response be in response.operation.refresh()?